stable/rabbitmq/values.yaml

```diff
@@ -146,6 +146,13 @@ rabbitmq:
   ## Configuration file content: advanced configuration
   ## Use this as additional configuraton in classic config format (Erlang term configuration format)
   advancedConfiguration: |-
+    # If you set LDAP with TLS/SSL enabled and you are using self-signed certificates, uncomment these lines.
+    #[
+    #{rabbitmq_auth_backend_ldap, [
+    #{ssl_options, [{verify, verify_none},
+    #{fail_if_no_peer_cert, false}]}
+    #]}
+    #].
```

Comment:
I feel like this should be in the config map itself under some if-else logic. I'm not sure about making the user "uncomment" settings since this would complicate things in distributions like Rancher.

Reply:
This configuration is very flexible, so there are many options that the user can set. 
I think it is better to left the user configure what he needs in each case.

In the README is suggested to look into the values.yml, I have noticed this approach in other charts.